umm-spec-lib/src/cmr/umm_spec/migration/version/subscription.clj

```diff
@@ -0,0 +1,18 @@
+(ns cmr.umm-spec.migration.version.subscription
+  "Contains functions for migrating between versions of the UMM subscription schema."
+  (:require
+   [clojure.string :as string]
+   [cmr.umm-spec.metadata-specification :as m-spec]
+   [cmr.umm-spec.migration.version.interface :as interface]))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+ ;;; subscription Migration Implementations
+
+
+(defmethod interface/migrate-umm-version [:subscription "1.0" "1.1"]
```

Comment:
Maybe add a down migration that log a reasonable error message rather than just an internal server error.